pkgs/development/haskell-modules/configuration-ghc-8.2.x.nix

```diff
@@ -71,4 +71,12 @@ self: super: {
   # https://github.com/aristidb/aws/issues/238
   aws = doJailbreak super.aws;
 
+  # There's no 8.2 testsuite
+  ghc-exactprint = dontCheck super.ghc-exactprint;
+
+  # Missing FlexibleContexts in test suite
```

Comment:
Is upstream aware of this issue? If they are, do you have a link to the upstream ticket so that we can see the current state of this bug?

Reply:
I'll open a ticket, thanks.